barman/backup.py

```diff
@@ -582,6 +582,12 @@ def delete_backup(self, backup, skip_wal_cleanup_if_standalone=True):
                 e.hook.script,
             )
 
+        # Update next rsync backup size information
+        if backup.config.reuse_backup:
```

Comment:
We should check that the backup is a rsync backup instead of a postgres backup. There is no need to check if reuse backup was set to link, as Israel mentioned

Reply:
This shoulf be checked from the `backup.info`